bin/cyhy-domain

```diff
@@ -27,38 +27,18 @@ import sys
 
 # Third-Party Libraries
 from docopt import docopt
-from validators import domain as valid_domain
 
 # cisagov Libraries
 from cyhy.core.common import *
 from cyhy.db import database
 from cyhy.util import util
 
-def parse_domains(domains):
-    parsed_domains = set()
-    invalid_domains = set()
-    for d in domains:
-        d = d.split("#", 1)[0]  # remove comments
-        d = d.strip()
-        if not d:
-            # skip blank lines
-            continue
-        if not valid_domain(d):
-            invalid_domains.add(d)
-        else:
-            parsed_domains.add(d)
-    
-    if invalid_domains:
-        print "ERROR - invalid domain(s):"
-        for d in invalid_domains:
-            print(d)
-        sys.exit(-1)
-    return parsed_domains
-
 
 def read_file(filename):
     f = open(filename, "r")
-    return parse_domains(f.readlines())
+    valid_domains, invalid_domains = util.parse_domains(f.readlines())
+    f.close()
```

Comment:
Would you please change this to a `with open(filename, "r") as ...` block?

Reply:
Done in https://github.com/cisagov/cyhy-core/pull/106/commits/a7ced5121bc2db91237c3344ef2a504704874b35.